c10/core/TensorOptions.h

```diff
@@ -5,6 +5,7 @@
 #include <c10/core/Layout.h>
 #include <c10/core/ScalarType.h>
 #include <c10/core/Device.h>
+//#include <c10/core/QScheme.h>
```

Comment:
ya, I don't think we should change TensorOptions.  We just might want to change how the mapping to Backend / TensorTypeId is done to include the dtype.  I'm guessing that's why you made quantized a device?  because there is no other way to change the lookup?

Reply:
I don't think dtype is enough for encoding all the quantization info, I have listed the options in the Implementation Notes: https://fb.quip.com/XGQnACVlKRdd

Reply:
cc @VitalyFedyunin 

I think we will be in serious trouble if we don't have *some* type which lets us conveniently build keyword arguments in C++. Maybe we don't have to call it TensorOptions, but I think we need something!